server/utils/queryHelpers/scopeGet.ts

```diff
@@ -319,24 +319,22 @@ getActivePermissions = async (
 		return currLevelIndex > prev ? currLevelIndex : prev;
 	}, defaultPermissionIndex);
 
-	const canAdminCommunity =
-		isSuperAdmin ||
-		scopeMemberData.find((member) => member.communityId && member.permissions === 'admin');
-
-	const canManageCommunity =
-		isSuperAdmin ||
-		canAdminCommunity ||
-		scopeMemberData.find((member) => member.communityId && member.permissions === 'manage');
-
-	const canEditCommunity =
-		isSuperAdmin ||
-		canAdminCommunity ||
-		scopeMemberData.find((member) => member.communityId && member.permissions === 'edit');
-
-	const canViewCommunity =
-		isSuperAdmin ||
-		canAdminCommunity ||
-		scopeMemberData.find((member) => member.communityId && member.permissions === 'view');
+	const scopeMember = (scopedMember: any, permissionLevel: string) => {
+		return (
+			permissionLevels.includes(permissionLevel) &&
+			scopedMember.find(
+				(member) => member.communityId && member.permissions === permissionLevel,
+			)
+		);
+	};
+
+	const member = scopeMember.bind(this, scopeMemberData);
```

Comment:
A note on `this`: If you're in an arrow function, and most of our functions are arrow functions, then `this` is basically useless — it can't be re-bound to anything! In this specific instance:

1. `this` is not defined for the containing arrow function `getActivePermissions`
2.  The enclosed arrow function `scopeMember` won't care about what you pass as the first argument to `bind`

It's completely harmless to reference `this` here, but it won't let you do anything. I would write:

```ts
scopeMember.bind(null, scopeMemberData)
```

this signals to the reader of this code that you don't expect `scopeMember` to do anything with its `this` value. Which makes sense, because it doesn't reference it! This is good practice even when working with non-arrow functions — if you're not going to use `this`, don't bind it forward!